### ISSUE TITLE
AvroIdlErrorReportSubmitter crashing when reporting another exception in this plugin

### ISSUE THREAD

**Issue by tmoschou** [opwvhk/avro-schema-support#267]
**Description**

When this plugin throws an internal exception, and when choosing to report the exception to this project, another exception is thrown inside opwvhk.intellij.avro_idl.AvroIdlErrorReportSubmitter / org.kohsuke.github.GitHubBuilder.

I presume neither stack traces are being reported automatically if the reporter is failing. I have included the stacktraces of both.

First exception thrown 
```
com.intellij.diagnostic.PluginException: Legacy configurable id calculation mode from localizable name will be used for configurable class opwvhk.intellij.avro_idl.editor.AvroIdlCodeStyleSettingsProvider. Please override getConfigurableId or getLanguage. [Plugin: net.sf.opk.avro-schema-support]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
	at com.intellij.diagnostic.PluginException.logPluginError(PluginException.java:112)
	at com.intellij.psi.codeStyle.CodeStyleSettingsProvider.getConfigurableId(CodeStyleSettingsProvider.java:71)
	at com.intellij.application.options.CodeStyleConfigurableWrapper.getId(CodeStyleConfigurableWrapper.java:112)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.getId(ConfigurableVisitor.java:97)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.lambda$findById$0(ConfigurableVisitor.java:37)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.find(ConfigurableVisitor.java:56)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.find(ConfigurableVisitor.java:63)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.find(ConfigurableVisitor.java:63)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.find(ConfigurableVisitor.java:46)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.findById(ConfigurableVisitor.java:37)
	at com.intellij.openapi.options.newEditor.SettingsEditor.<init>(SettingsEditor.java:387)
	at com.intellij.openapi.options.newEditor.SettingsDialog.<init>(SettingsDialog.java:93)
	at com.intellij.openapi.options.newEditor.SettingsDialogFactory.create(SettingsDialogFactory.kt:45)
	at com.intellij.ide.actions.ShowSettingsUtilImpl$showSettingsDialog$4.invokeSuspend(ShowSettingsUtilImpl.kt:187)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:34)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:100)
	at com.intellij.openapi.application.impl.DispatchedRunnable.run(DispatchedRunnable.kt:42)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:239)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:25)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:221)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$0$0(NonBlockingFlushQueue.kt:326)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:897)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$0(NonBlockingFlushQueue.kt:325)
	at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:294)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1(NonBlockingFlushQueue.kt:322)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.tryRunWriteIntentReadAction(NestedLocksThreadingSupport.kt:690)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent(NonBlockingFlushQueue.kt:320)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.flushNow(NonBlockingFlushQueue.kt:299)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.FLUSH_NOW$lambda$0(NonBlockingFlushQueue.kt:167)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:750)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:664)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:517)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0$0$0(IdeEventQueue.kt:333)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1065)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:109)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1065)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0(IdeEventQueue.kt:331)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:371)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```

When reporting this exception, another is thrown
```
java.lang.NoSuchFieldError: Class com.fasterxml.jackson.databind.PropertyNamingStrategy does not have member field 'com.fasterxml.jackson.databind.PropertyNamingStrategy SNAKE_CASE'
	at org.kohsuke.github.GitHubClient.<clinit>(GitHubClient.java:81)
	at org.kohsuke.github.GitHub.<init>(GitHub.java:137)
	at org.kohsuke.github.GitHubBuilder.build(GitHubBuilder.java:509)
	at opwvhk.intellij.avro_idl.AvroIdlErrorReportSubmitter.connectToGitHub(AvroIdlErrorReportSubmitter.java:198)
	at opwvhk.intellij.avro_idl.AvroIdlErrorReportSubmitter.lambda$submit$1(AvroIdlErrorReportSubmitter.java:89)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:239)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:25)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:201)
	at com.intellij.openapi.application.impl.AppImplKt$runnableUnitFunction$1.invoke(appImpl.kt:124)
	at com.intellij.openapi.application.impl.AppImplKt$runnableUnitFunction$1.invoke(appImpl.kt:124)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.doRunWriteIntentReadAction(NestedLocksThreadingSupport.kt:737)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runPreventiveWriteIntentReadAction(NestedLocksThreadingSupport.kt:711)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.runWriteIntentReadAction(NestedLocksThreadingSupport.kt:665)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:1048)
	at com.intellij.openapi.application.impl.ApplicationImpl$7.lambda$run$0(ApplicationImpl.java:602)
	at com.intellij.concurrency.ThreadContext.installThreadContext(threadContext.kt:305)
	at com.intellij.openapi.application.impl.ApplicationImpl$7.run(ApplicationImpl.java:600)
	at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:167)
	at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:167)
	at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:173)
	at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:167)
	at com.intellij.util.concurrency.ContextRunnable.lambda$run$0(ContextRunnable.java:26)
	at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:294)
	at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:25)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$0$0(NonBlockingFlushQueue.kt:326)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:897)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1$0(NonBlockingFlushQueue.kt:325)
	at com.intellij.concurrency.ThreadContext.resetThreadContext(threadContext.kt:294)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent$lambda$1(NonBlockingFlushQueue.kt:322)
	at com.intellij.platform.locking.impl.NestedLocksThreadingSupport.tryRunWriteIntentReadAction(NestedLocksThreadingSupport.kt:690)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.runNextEvent(NonBlockingFlushQueue.kt:320)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.flushNow(NonBlockingFlushQueue.kt:299)
	at com.intellij.openapi.application.impl.NonBlockingFlushQueue.FLUSH_NOW$lambda$0(NonBlockingFlushQueue.kt:167)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:750)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:664)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:517)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0$0$0(IdeEventQueue.kt:333)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:1065)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:109)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1065)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$0(IdeEventQueue.kt:331)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:371)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
```
Alternatively I also get
```
java.lang.NoClassDefFoundError: Could not initialize class org.kohsuke.github.GitHubClient
	at org.kohsuke.github.GitHub.<init>(GitHub.java:137)
	at org.kohsuke.github.GitHubBuilder.build(GitHubBuilder.java:509)
	at opwvhk.intellij.avro_idl.AvroIdlErrorReportSubmitter.connectToGitHub(AvroIdlErrorReportSubmitter.java:198)
	at opwvhk.intellij.avro_idl.AvroIdlErrorReportSubmitter.lambda$submit$1(AvroIdlErrorReportSubmitter.java:89)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:239)
...
```

Should note I am using 2025.3 EAP.

Looks like for some reason, the Jackson version used at runtime / by the classloader is an older version, then the one intended.

**To Reproduce**

Steps to reproduce the behaviour:
1. Install IntelliJ IDEA 2025.3 EAP IU-253.28086.51, 
2. Install plugin 243.0.1, etc
3. reopen intellij/project (exception usually happens on boot) 

**Expected behaviour**

Plugin doesn't throw internal exceptions.

**Screenshots**

<img width="1571" height="534" alt="Image" src="https://github.com/user-attachments/assets/f74611f4-e066-4af5-ac6c-6d4451e831ae" />

<img width="1592" height="527" alt="Image" src="https://github.com/user-attachments/assets/dfb84e25-c4f0-4d7f-9cd7-8070ca005b14" />

<img width="1445" height="966" alt="Image" src="https://github.com/user-attachments/assets/b4d8988c-7e2b-446a-a057-99fdabad05eb" />

**Versions:**
```
IntelliJ IDEA 2025.3 EAP
Build #IU-253.28086.51, built on November 3, 2025
Source revision: 9ea59ce91cbd8
Licensed to <REDACTED>
Runtime version: 21.0.8+9-b1163.69 aarch64 (JCEF 137.0.17)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.lwawt.macosx.LWCToolkit
macOS 15.7.1
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 8192M
Cores: 12
Metal Rendering is ON
Registry:
  ide.experimental.ui=true
  kotlin.scripting.index.dependencies.sources=true
  terminal.new.ui.reworked=true
Non-Bundled Plugins:
  com.intellij.notebooks.core (253.28086.52)
  com.intellij.debugger.collections.visualizer (253.28086.52)
  JavaScript (253.28086.55)
  org.antlr.intellij.plugin (1.25)
  training (253.28086.57)
  JavaScriptDebugger (253.28086.55)
  org.intellij.plugins.hcl (253.28086.9)
  Subversion (253.28086.52)
  HtmlTools (253.28086.55)
  PythonCore (253.28086.51)
  org.toml.lang (253.28086.52)
  org.jetbrains.plugins.go-template (253.28086.9)
  net.sf.opk.avro-schema-support (243.0.1)
  com.intellij.javaee (253.28086.52)
  com.intellij.spring (253.28086.52)
  EclipseCodeFormatter (23.5.241.000.0-Eclipse_2024-09)
  org.intellij.scala (2025.3.15)
  com.intellij.properties (253.28086.52)
  com.intellij.velocity (253.28086.52)
  com.haulmont.jpab (253.28086.51)
  AntSupport (253.28086.51)
  com.intellij.spring.debugger (253.28086.9)
  com.anthropic.code.plugin (0.1.12-beta)
  CheckStyle-IDEA (5.114.0)
  com.intellij.spring.websocket (253.28086.51)
  org.jetbrains.plugins.go (253.28086.51)
  ru.artyushov (1.6.0)
  gherkin (253.28086.9)
  Pythonid (253.28086.51)
  com.intellij.tasks (253.28086.52)
  com.intellij.tasks.timeTracking (253.28086.52)
  com.intellij.aqua (253.28086.9)
  cucumber-java (253.28086.51)
  intellij.jupyter (253.28086.52)
  Docker (253.28086.57)
  org.asciidoctor.intellij.asciidoc (0.45.1)
  com.intellij.reactivestreams (253.28086.51)
  intellij.webpack (253.28086.55)
  com.intellij.react (253.28086.55)
  intellij.vitejs (253.28086.52)
  NodeJS (253.28086.52)
  com.dmarcotte.handlebars (253.28086.9)
  org.jetbrains.plugins.vue (253.28086.57)
  tslint (253.28086.52)
  org.jetbrains.plugins.node-remote-interpreter (253.28086.52)
  com.deadlock.scsyntax (253.28086.52)
  com.intellij.plugins.webcomponents (253.28086.52)
  com.haulmont.rcb (253.28086.9)
  intellij.prettierJS (253.28086.52)
  org.mapstruct.intellij (1.9.1)
  intellij.nextjs (253.28086.52)
  Karma (253.28086.52)
  com.intellij.mcpServer (253.28086.55)
  mobi.hsz.idea.gitignore (4.5.6)
  org.jetbrains.security.package-checker (253.28086.57)
  com.github.copilot (1.5.60-243)
  dev.nx.console (1.54.0)
Kotlin: 253.28086.51-IJ
```

**Additional context**

**Comment by opwvhk** [comment]
Thank you for reporting the error reporter crash!

I'll create a new version soon to fix both issues.
